test/runScript-mock-findBin.spec.js

```diff
@@ -38,17 +29,16 @@ const packageJSON = {
   'lint-staged': {}
 }
 
-describe.only('runScript with absolute paths', () => {
+describe('runScript with absolute paths', () => {
   beforeEach(() => {
     mockFn.mockReset()
     mockFn.mockImplementation(() => Promise.resolve(true))
   })
 
   it('can pass `gitDir` as `cwd` to `execa()` when git is called via absolute path', async () => {
     const res = runScript(['git add'], ['test.js'], packageJSON, { gitDir: '../' })
-
     const taskPromise = res[0].task()
-    expect(taskPromise).toBeAPromise()
+    expect(taskPromise).toBeInstanceOf(Promise)
     await taskPromise
     expect(mockFn.mock.calls.length).toEqual(1)
```

Comment:
This can be simplified:
```js
expect(mockFn).toHaveBeenCalledTimes(1)
expect(mockFn).toHaveBeenCalledWith('/usr/local/bin/git', ['add', 'test.js'], { cwd: '../' })
```